packages/web-react/src/components/Grid/Grid.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import Grid from './Grid';
+import { SpiritGridProps } from '../../types';
+
+const gridVariants = [1, 2, 3, 4, 6, 12];
+
+export default {
+  title: 'Components/Grid',
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    mobile: gridVariants,
+    tablet: gridVariants,
+    desktop: gridVariants,
+  },
+};
+
+const exampleGridStyle = {
+  height: '2rem',
+  boxSizing: 'border-box',
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  color: '#fff',
+  background: '#0b1c21',
+};
+
+const Template = (args: SpiritGridProps) => <Grid {...args} />;
+
+export const EqualColumns = Template.bind({});
+EqualColumns.args = {
+  children: (
+    <>
+      <div style={exampleGridStyle}></div>
+      <div style={exampleGridStyle}></div>
+      <div style={exampleGridStyle}></div>
```

Comment:
I'm happy all these inline styles are just an example and they don't make it to production 😄.